openpype/hosts/substancepainter/plugins/create/create_textures.py

```diff
@@ -34,6 +34,18 @@ def create(self, subset_name, instance_data, pre_create_data):
         if not substance_painter.project.is_open():
             raise CreatorError("Can't create a Texture Set instance without "
                                "an open project.")
+        # Transfer settings from pre create to instance
+        creator_attributes = instance_data.setdefault(
+            "creator_attributes", dict())
+        for key in [
+            "exportPresetUrl",
+            "exportFileFormat",
+            "exportSize",
+            "exportPadding",
+            "exportDilationDistance"
+        ]:
+            if key in pre_create_data:
+                creator_attributes[key] = pre_create_data[key]
```

Comment:
Oh yeah baby, I'd love it if this works! Since it's relatively clean and isolated where the logic is. Does it work? 😮

Reply:
it does work and the parameter is propagated to the instance.